cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraTablesCreator.java

```diff
@@ -1,343 +1,394 @@
+			}
+			return;
+		}
+		
+		CreateTableWithOptions create = SchemaBuilder.createTable(settings.getKeyspace(), tableName).ifNotExists()
+				.withPartitionKey(INSTANCE_ID, DataTypes.UUID)
+				.withPartitionKey(PARENT_ID, DataTypes.TEXT)
+				.withPartitionKey(START_DATE, DataTypes.DATE)
+				.withClusteringColumn(START_TIME, DataTypes.TIME)
+				.withClusteringColumn(ID, DataTypes.TEXT)
+				.withColumn(ROOT, DataTypes.BOOLEAN)
+				.withColumn(NAME, DataTypes.TEXT)
+				.withColumn(TYPE, DataTypes.TEXT)
+				.withColumn(EVENT_BATCH, DataTypes.BOOLEAN)
+				.withColumn(END_DATE, DataTypes.DATE)
+				.withColumn(END_TIME, DataTypes.TIME)
+				.withColumn(SUCCESS, DataTypes.BOOLEAN)
+				.withColumn(EVENT_COUNT, DataTypes.INT)
+				.withColumn(EVENT_BATCH_METADATA, DataTypes.BLOB)
+				.withClusteringOrder(START_TIME, ClusteringOrder.ASC)
+				.withClusteringOrder(ID, ClusteringOrder.ASC);
+		
+		exec.executeQuery(create.asCql(), true);
+		logger.info("Table '{}' has been created", tableName);
+	}
+	
+	public void createTestEventsChildrenDatesTable() throws IOException
+	{
+		String tableName = settings.getTestEventsChildrenDatesTableName();
+		if (isTableExists(tableName))
+			return;
+		
+		CreateTableWithOptions create = SchemaBuilder.createTable(settings.getKeyspace(), tableName).ifNotExists()
+				.withPartitionKey(INSTANCE_ID, DataTypes.UUID)
+				.withPartitionKey(PARENT_ID, DataTypes.TEXT)
+				.withClusteringColumn(START_DATE, DataTypes.DATE)
+				.withClusteringOrder(START_DATE, ClusteringOrder.ASC);
+		
+		exec.executeQuery(create.asCql(), true);
+		logger.info("Table '{}' has been created", tableName);
+	}
+	
+	public void createTestEventsMessagesTable() throws IOException
+	{
+		String tableName = settings.getTestEventsMessagesTableName();
+		if (isTableExists(tableName))
+			return;
+		
+		CreateTable create = SchemaBuilder.createTable(settings.getKeyspace(), tableName).ifNotExists()
+				.withPartitionKey(INSTANCE_ID, DataTypes.UUID)
+				.withPartitionKey(TEST_EVENT_ID, DataTypes.TEXT)
+				.withClusteringColumn(ID, DataTypes.TIMEUUID)
+				.withColumn(MESSAGE_IDS, DataTypes.setOf(DataTypes.TEXT));
+//				.withClusteringColumn(MESSAGE_IDS, DataTypes.frozenSetOf(DataTypes.TEXT));
+		
+		exec.executeQuery(create.asCql(), true);
+		logger.info("Table '{}' has been created", tableName);
+	}
+	
+	public void createMessagesTestEventsTable() throws IOException
+	{
+		String tableName = settings.getMessagesTestEventsTableName();
+		if (isTableExists(tableName))
+			return;
+		
+		CreateTable create = SchemaBuilder.createTable(settings.getKeyspace(), tableName).ifNotExists()
+				.withPartitionKey(INSTANCE_ID, DataTypes.UUID)
+				.withPartitionKey(MESSAGE_ID, DataTypes.TEXT)
+				.withClusteringColumn(TEST_EVENT_ID, DataTypes.TEXT)
+				.withColumn(BATCH_ID, DataTypes.TEXT);
+		
+		exec.executeQuery(create.asCql(), true);
+		logger.info("Table '{}' has been created", tableName);
```

Comment:
Looks like this portion can be extracted into separate method to be called everywhere.